### ISSUE TITLE
Qt on Homebrew no longer includes QtWebkit

### ISSUE THREAD

**Issue by mathias** [thoughtbot/capybara-webkit#1053]
These instructions don't talk about how to get QtWebkit: https://github.com/thoughtbot/capybara-webkit/wiki/Installing-Qt-and-compiling-capybara-webkit#homebrew

I think this is the cause of #1052 -- I'm seeing that issue on bundle, but when I started investigating, I found that my new laptop does not have QtWebkit anywhere after `brew install qt@5.5` per the above link.

Is there a known solution to this?

**Comment by mathias** [comment]
Some additional information for diagnosing: I have 
`export PATH="$(brew --prefix qt@5.5)/bin:$PATH"`
in my `.bashrc` files.
`gem install capybara-webkit` fails for 1.14.0 on Ruby 2.3.6.

**Comment by etagwerker** [comment]
@mathias What does `which qmake` output? 

One thing I've noticed is that adding `export PATH="$(brew --prefix qt@5.5)/bin:$PATH"` to your `.bashrc` is not enough. You need to run `source .bashrc` or open a new terminal (for it to load it automatically)

**Comment by mathias** [comment]
Yep; it has already been in there according to my dotfiles git history, since before I received this work laptop. I only ran into this issue when I tried to upgrade Ruby version in projects locally and our gem cache in /vendor not longer had the gem. The path in my terminals all start with:

```
$ echo $PATH
/usr/local/opt/qt@5.5/bin:/Users/mgauger/.rbenv/shims:...etc
```

**Comment by mathias** [comment]
```
$ which qmake
/usr/local/opt/qt@5.5/bin/qmake
```

**Comment by mathias** [comment]
Except when trying to bundle / gem install looks like

```
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /Users/mgauger/.rbenv/versions/2.3.6/lib/ruby/gems/2.3.0/gems/capybara-webkit-1.14.0
/Users/mgauger/.rbenv/versions/2.3.6/bin/ruby -r ./siteconf20180116-91066-w41knz.rb extconf.rb
Info: creating stash file /Users/mgauger/.rbenv/versions/2.3.6/lib/ruby/gems/2.3.0/gems/capybara-webkit-1.14.0/.qmake.stash
cd src/ && ( test -e Makefile.webkit_server || /Users/mgauger/anaconda3/bin/qmake
/Users/mgauger/.rbenv/versions/2.3.6/lib/ruby/gems/2.3.0/gems/capybara-webkit-1.14.0/src/webkit_server.pro -o Makefile.webkit_server ) &&
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f Makefile.webkit_server
Project ERROR: No QtWebKit installation found. QtWebKit is no longer included with Qt 5.6, so you may need to install it separately.
make: *** [sub-src-webkit_server-pro-make_first-ordered] Error 3
Command 'make ' failed
```

**Comment by mathias** [comment]
That must be it:

```
$ qmake --version
QMake version 3.0
Using Qt version 5.6.2 in /Users/mgauger/anaconda3/lib
$ /usr/local/opt/qt@5.5/bin/qmake --version
QMake version 3.0
Using Qt version 5.5.1 in /usr/local/Cellar/qt@5.5/5.5.1_1/lib
```

**Comment by HHGB12** [comment]
Thanks a lot  👌 @etagwerker  for someone else who `loves` screenshot have a look on it

`brew install qt@5.5 `
=> `Warning: qt@5.5 5.5.1_1 is already installed`

$ `echo 'export PATH="$(brew --prefix qt@5.5)/bin:$PATH"' >> ~/.bashrc `
$ `which qmake`
=> ⚠️ qmake not found

$ `source .bashrc `
$ `which qmake `
=> ✅ /usr/local/opt/qt@5.5/bin/qmake


<img width="866" alt="capture d ecran 2018-02-17 a 14 34 14" src="https://user-images.githubusercontent.com/24680535/36341525-e2a114a2-13ef-11e8-8317-32c58a693084.png">

**Comment by miguelaorz2018** [comment]
Mathias did you solve the problem?

**Comment by mathias** [comment]
Yes, it seems like I had to make sure I wasn't using the QMake from Anaconda (A Python data science package manager / version manager.)

**Comment by syedrakib** [comment]
I am still getting the `Project ERROR: No QtWebKit installation found. QtWebKit is no longer included with Qt 5.6, so you may need to install it separately.` error when i try to do `bundle install` even though `qmake` is in its path all well

```
➜ $ qmake --version
QMake version 3.1
Using Qt version 5.12.0 in /usr/local/Cellar/qt/5.12.0/lib
➜  $ /usr/local/opt/qt/bin/qmake --version
QMake version 3.1
Using Qt version 5.12.0 in /usr/local/Cellar/qt/5.12.0/lib
```

**Comment by zsemenenko** [comment]
had the same issue, found decription in "webkit_server.pro" file :
  ```
qtHaveModule(webkitwidgets) {
    QT += webkitwidgets
   } else {
      error("No QtWebKit installation found. QtWebKit is no longer included with Qt 5.6, so you may need to install it separately.")
   }
```
so, u need to install "webkitwidgets", for debian it was found in package "libqt5webkit5-dev"
